File: CSharp/Shop/Ui/CLI/UiTextCreator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Shop.Ui.CLI {
    using Items;
    using Customers;
    public interface UiTextCreator {
        public string ConstructMainMenu();
        public string ConstructShopMenu(List<ItemBatch> items);
        public string ConstructCustomerMenu(Customer c);
        public string ConstructCustomerBuyMenu(List<ItemBatch> shopItems);
        public string ConstructCustomerRefundMenu(List<Item> items);
    }
}

File: Java/TddShop/src/Ui/UiTextCreator.java
package Ui;

import java.util.List;
import Item.*;
import Customer.*;

public interface UiTextCreator {
    public String ConstructMainMenu();
    public String ConstructShopMenu(List<ItemBatch> items);

    public String ConstructCustomerMenu(Customer c);
    public String ConstructCustomerBuyMenu(List<ItemBatch> shopItems);
    public String ConstructCustomerRefundMenu(List<Item> customerItems);
}
